packages/botonic-react/src/webchat/use-storage-state-hook.js

```diff
@@ -0,0 +1,9 @@
+import { useLocalStorage, useSessionStorage } from 'react-use-storage'
+
+export function useStorageState(storage, storageKey) {
+  const [botonicLocalState, saveLocalState] = useLocalStorage(storageKey)
```

Comment:
can you move these 2 lines right before they're used? or they're hooks that cannot be within "ifs"?